src/components/directory-tree-view.js

```diff
@@ -98,6 +98,12 @@ const DirectoryTreeView = ({ treeViewUuid, mapData, onContextMenu }) => {
             <TreeItem
                 key={node.elementUuid}
                 nodeId={node.elementUuid}
+                onIconClick={() => {
+                    toggleDirectory(node.elementUuid);
+                }}
+                onLabelClick={() => {
+                    openDirectory(node.elementUuid);
```

Comment:
- add a `handleLabelClick` function which replace `handleSelect` and call it here like :
`handleLabelClick(node.elementUuid, !expandedRef.current.includes(node.elementUuid)`
- remove `openDirectory` function
- when `handleSelect` is actually called, rename the function : 
`function handleContextMenuClick(event, nodeId) {
        handleLabelClick(nodeId, false);
        onContextMenu(event, nodeId);
    }`